src/routes/InventoryDetail.js

```diff
@@ -16,6 +17,7 @@ import '@redhat-cloud-services/frontend-components-inventory-insights/index.css'
 import '@redhat-cloud-services/frontend-components-inventory-vulnerabilities/dist/cjs/index.css';
 import { SystemCvesStore } from '@redhat-cloud-services/frontend-components-inventory-vulnerabilities/dist/cjs/SystemCvesStore';
 import { SystemAdvisoryListStore } from '@redhat-cloud-services/frontend-components-inventory-patchman/dist/esm';
+import { SystemPackageListStore } from '@redhat-cloud-services/frontend-components-inventory-patchman/dist/esm';
```

Comment:
These lines can be merged together
```suggestion
import { SystemAdvisoryListStore, SystemPackageListStore  } from '@redhat-cloud-services/frontend-components-inventory-patchman/dist/esm';
```